openreview/conference/templates/areachairWebfield.js

```diff
@@ -801,10 +805,28 @@ var registerEventHandlers = function() {
   });
 
   $('#group-container').on('click', 'a.collapse-btn', function(e) {
+    var paperIdArray = $(this).attr('href').split("-");//href attribute is in format of #{paperId}-reviewers
+    paperIdArray.pop(); //
+    var paperId = paperIdArray.join('-').substring(1);
+    var paperNumber = reviewerSummaryMap[Object.keys(reviewerSummaryMap).filter(function (p) {
```

Comment:
This would be better separated into 2 statements with an intermediate var for `Object.keys...`